examples/IrisClassificationTorchScript/README.md

```diff
@@ -1,5 +1,9 @@
-In this example, we train a Pytorch Lightning model to classify iris based on height and width of sepal and petal. Then, we convert the model to TorchScript 
-and serve the scripted model in TorchServe from Mlflow. 
+# Deploying Iris Classification using torchserve
```

Comment:
@karthik-77 It might help to pass the index_to_name.json as extra file during creating a new deployment

Reply:
made the suggested changes